emlparser/convert_outlook/outlookmsgfile.py

```diff
@@ -12,18 +12,20 @@
 # https://msdn.microsoft.com/en-us/library/ee157583(v=exchg.80).aspx
 # https://blogs.msdn.microsoft.com/openspecification/2009/11/06/msg-file-format-part-1/
 
-import compoundfiles
 import email.message
 import email.parser
 import email.policy
-import re
 import os
+import re
 import sys
 
-from email.utils import parsedate_to_datetime, formatdate, formataddr
-from compressed_rtf import decompress
+from builtins import isinstance
```

Comment:
you don't need to import a builtin, you can just call `isinstance` in the code

Reply:
I hate when VSCode does this.. I didn't add that, but good catch!